src/screens/login/LoginScreen.js

```diff
@@ -25,8 +30,29 @@ export const LoginScreen = () => {
         passwordError: 'Password is required',
       }));
     }
-    if(!errors.emailError && !errors.passwordError){
-        //If there's no errors we can fetch the login api and check if the credentials are correct
+    if (errors.emailError === '' && errors.passwordError === '') {
+      setIsLoading(true);
+      quizAPI
+        .post('/login', {
+          TableName: 'Users',
+          Email: values.email,
+          Password: values.password,
+        })
+        .then(function (response) {
+          if (response.data === 'Wrong Email or Password.') {
+            setErrors(prevState => ({
+              ...prevState,
+              credentialError: response.data,
+            }));
+          } else {
+            // Temporary alert until navigation is handled
+            alert('Login Successfull');
+          }
+        })
+        .catch(function (error) {
+          console.log('Error: ', error);
```

Comment:
we can use console.error() to show a better logging

Reply:
Changed it to use console.error()